### PR TITLE
Add react-ui-tracking

### PR DESCRIPTION
**Pull request by aribouius**
- Introduces a new child repo `react-ui-tracking` for components related to event tracking.
- Adds components taken from AG.js (`Tracker`, `Tagging`, `PageView`) which have been modified to be `redux` agnostic while still supporting similar functionality.

### CODE REVIEW

File: packages/react-ui-tracking/src/Tracker.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      this._pageView = { ...props }
+      this.update().then(() => {
+        this.props.pageView(this._pageView)
+        this._pageView = null
+      })
+    }
+  }
+
+  render() {
+    return Array.isArray(this.props.children) ? (
```
Comment (lfender6445) [inline packages/react-ui-tracking/src/Tracker.js]: can you elaborate on this piece? why wrap with a div?

Comment (aribouius) [reply packages/react-ui-tracking/src/Tracker.js]: because you can't render an array of children (yet).

**Comment by lfender6445** [comment]
I'm 👍  and we can extend as needed but mind adding a readme? Whats left?

**Comment by ninomichaels** [comment]
LGTM 👍 , agree with Luke though, maybe some basic usage examples?

**Comment by aribouius** [comment]
@lfender6445 @lfender6445 thanks guys I need to get this out so I can push a separate PR on `rent-js`, but I'll add some docs today.

**Pull request merged**